rules-tests/TypeDeclaration/Rector/ArrowFunction/AddArrowFunctionReturnTypeRector/Fixture/return_array.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\TypeDeclaration\Rector\ArrowFunction\AddArrowFunctionReturnTypeRector\Fixture;
```

Comment:
The "Add*Rector" naming is used for configurable rules to add spefic known types, e.g. during Symfony upgrade, but this rule is not configurable.

Name like `ArrowFunctionReturnTypeFrom*Rector` would be better.

Reply:
There is existing `AddArrowFunctionReturnTypeRector` rule , it is moving the fixture as functionality covered already in `AddArrowFunctionReturnTypeRector`

Reply:
I see :+1: